_posts/2019-09-30-introducing-arrow-flight.md

```diff
@@ -0,0 +1,278 @@
+---
+layout: post
+title: "Introducing Apache Arrow Flight: A Framework for Fast Data Transport"
+description: "This post introduces Arrow Flight, a framework for building high
+performance data services. We have been building Flight over the last 18 months
+and are looking for developers and users to get involved."
+date: "2019-10-13 00:00:00 -0600"
+author: Wes McKinney
+categories: [application]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+Over the last 18 months, the Apache Arrow community has been busy designing and
+implementing **Flight**, a new general-purpose client-server framework to
+simplify high performance transport of large datasets over network interfaces.
+
+Flight initially is focused on optimized transport of the Arrow columnar format
+(i.e. "Arrow record batches") over [gRPC][1], Google's popular HTTP/2-based
+general-purpose RPC library and framework. While we have focused on integration
+with gRPC, as a development framework Flight is not intended to be exclusive to
+gRPC.
+
+In the 0.15.0 Apache Arrow release, we have ready-to-use Flight implementations
+in C++ (with Python bindings) and Java. These libraries are suitable for beta
+users who are comfortable being on the bleeding edge while we continue to
```

Comment:
Let's remove "bleeding edge". I agree that it is beta but several Dremio users are already finding a bunch of value with it and it has gone through two releases.

Reply:
done